scss/_patterns_navigation.scss

```diff
@@ -161,18 +154,28 @@
             border-bottom: 1px solid $color-mid-light;
           }
         }
-
-        @media (min-width: $breakpoint-navigation-threshold + 1) {
-          padding: $sp-small ($sp-large - $sp-xx-small);
-        }
       }
 
+      .p-navigation__link > a,
       > a {
-        // For when this class is applied to something that contains <a>s, e.g.:
-        // <li class="p-navigation__list"><a></a></li>
-        @media (max-width: $breakpoint-navigation-threshold) {
-          color: $color-dark;
-          display: block;
+        // For links nested in navigation list, e.g:
+        // <ul class="p-navigation__links">
+        //  <li class="p-navigation__link"><a></a></li>
+        // </ul>
+        //
+        // and for links nested directly in navigation, e.g:
+        // <div class="p-navigation__links">
+        //   <a class="class="p-navigation__link"></a>
+        // </div>
+
+        color: $color-dark;
+        font-size: .875rem;
+        padding: .84375rem $sp-x-small;  // 0.84375rem = (48px [height] - 21px [line-height]) / 2
```

Comment:
Custom padding of 13,5px (.84375rem) needs to be used to add up from 21px line-height to 48px desired height.